### PR TITLE
fix user menu avatar line height

### PR DESCRIPTION
**Pull request by ektasawant**
Fixes [419](https://github.com/brightlayer-ui/angular-component-library/issues/419) .

#### Changes proposed in this Pull Request:
- Fix user menu avatar line height

#### Screenshots:
Screenshot 2022-04-07 at 5 36 02 PM

#### To Test:
Angular showcase demo
- /blui-components/data-display-components

### CODE REVIEW

**Review by emclaug2 — APPROVED** [review]
Can you update the changelog as well? Looks good to me.

**Comment by emclaug2** [comment]
Also can you update the package version so we can publish a beta when this gets merged in?

**Comment by ektasawant** [comment]
> Also can you update the package version so we can publish a beta when this gets merged in?

Updated. Please check.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Change Log
 
+## v7.0.1 (Not yet published)
+
+### Fixed
+
```
Comment (emclaug2) [inline CHANGELOG.md:6]: Small verbiage suggestion: 

> Fixed avatar line-height in `<blui-user-menu>` component ([#419](https://github.com/brightlayer-ui/angular-component-library/issues/419))

**Review by emclaug2 — APPROVED** [review]

**Review by emclaug2 — APPROVED** [review]

**Pull request merged**